front-end-development/front-end-development-quiz.md

```diff
@@ -495,3 +495,12 @@ console.log(currencies);
 - [ ] This will stop the CCS being loaded on slow data connections.
 
 [Source: StackOverflow](https://stackoverflow.com/a/46750893)
+
+#### Q52. The flex property is often applied to flex items with a value of 1. Using flex: 1 is a shorthand - what does it unpack to?
+
+- [ ] flex: 0 0 100;
+- [ ] flex: 1 0 0;
+- [ ] flex: 0 0 auto;
+- [x] flex: 1 1 auto;
```

Comment:
This one is incorrect, `flex:1` is equivalent to `flex: 1 1 0`. Check the link you provided.
`flex: 1 1 auto` comes from `flex: auto` : https://css-tricks.com/almanac/properties/f/flex/#flex-auto

Did you take a screenshot of the original question? It seems that options have no correct answer to the provided question.

Reply:
I have a screenshot from the Linkedin skill quiz.

![Capture3](https://user-images.githubusercontent.com/84046762/140805737-e274db4b-9409-4c3f-9eee-bd214de07b34.JPG)